docs/quality-assurance-test-cases.md

```diff
@@ -174,6 +174,31 @@ This document contains some examples of test cases for each feature implemented
 | Logged in user tries to complete a task from a request which is not in the ACCEPTED state (as a mentor or as a mentee) | Fail |
 
 
+### Tasks
+
+#### Create
+
+**Service:** POST /mentorship_relation/{relation_id}/task
+
+|  Test Case                                                                                | Outcome |
+| ----------------------------------------------------------------------------------------- |-------- |
+| Create a task for a relation, in the accepted state, between logged user and another user | Success |
+| Creating a task without a description (either empty or not in the request body at all)    | Fail    |
+| Create a task when a logged user is not involved in the relation                          | Fail    |
+| Create a task if relation state is different than accepted                                | Fail    |
+
+#### List All Tasks
+
+**Service:** GET /mentorship_relation/{request_id}/task
+
+| Test Case        | Outcome |
+| -------------- | ----------- |
+| Logged in user requests list all task,or out of all task,for a mentorship relation those which involves the user (as a mentor or as a mentee) | Success |
```

Comment:
@khushishikhu I don't quite understand the sentence, what do you mean here?

Reply:
@isabelcosta  i mean that the task will be successful when the logged in user which is either a mentor or a mentee requests to list it's all tasks for a mentorship relation . This is what i am getting that should be written over here, if there is still something inappropriate please let me know or guide me as well. Thank you